build.gradle

```diff
@@ -163,15 +163,11 @@ clean {
 dependencies {
     api platform('com.exactpro.th2:bom:4.2.0')
 
-    implementation 'com.exactpro.th2:common:5.3.0-json-raw-body-5144283961-5ae02cd-SNAPSHOT'
+    implementation 'com.exactpro.th2:common:5.3.0-json-raw-body-5146306196-dadcaea-SNAPSHOT'
     implementation 'com.exactpro.th2:common-utils:2.1.0-transport-protocol-5144338850-92237a3-SNAPSHOT'
     implementation 'org.slf4j:slf4j-api'
 
-    implementation group: 'org.jetbrains.kotlin', name: 'kotlin-stdlib-jdk8', version: kotlin_version
-    implementation group: 'org.jetbrains.kotlin', name: 'kotlin-reflect', version: kotlin_version
-    implementation group: 'io.github.microutils', name: 'kotlin-logging', version: '1.7.9'
```

Comment:
Did we move that into a common or we simply don't use it here? If we moved it I don't think that is a good idea to have logging as an API dependency

Reply:
Maybe I don't know something, but we doesn't use these library directly in code or plugin add them itself